build.sh

```
@@ -0,0 +1,96 @@
+cp -r docs/* temp_docs/
+# Copy root files needed for Jekyll
+cp _config.yml temp_docs/
+cp -r _layouts temp_docs/ 2>/dev/null || true
+cp -r _includes temp_docs/ 2>/dev/null || true
+cp -r _data temp_docs/ 2>/dev/null || true
+cp -r _sass temp_docs/ 2>/dev/null || true
+cp -r assets temp_docs/ 2>/dev/null || true
+
+# Build from temp_docs directory
+echo "🏗️  Building Jekyll site from docs/ structure..."
+cd temp_docs
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)